File: payloads/python/read_sys_hardware.py
import os
found = []
for root, dirs, files in os.walk("/sys"):
    for f in files[:3]:
        try:
            found.append(open(os.path.join(root, f)).read()[:50])
        except:
            pass
if not found:
    raise RuntimeError("no /sys files readable - sandbox blocked access")
print("ESCAPED:", found)

File: payloads/java/read_sys_hardware.java
import java.nio.file.Files;
import java.nio.file.Path;
import java.nio.file.Paths;
import java.util.stream.Stream;

public class Main {
    public static void main(String[] args) throws Exception {
        try (Stream<Path> paths = Files.walk(Paths.get("/sys"), 2)) {
            paths.forEach(System.out::println);
        }
        System.out.println("ESCAPED");
    }
}
